css-pseudo-classes/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Pseudo Classes</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1>Hover Button</h1>
+    <form>
+      <label for="button"></label>
```

Comment:
If a `label` element is empty, then it is unnecessary. I appreciate the thought of including a `label` with the form control element, but in this case, it can be removed